boardswarm/src/dfu.rs

```diff
@@ -49,7 +49,7 @@ pub async fn start_provider(name: String, server: Server) {
                     continue;
                 };
 
-                let name = if let Some(model) = device.property("ID_MODEL") {
+                let name = if let Some(model) = device.property("ID_MODEL_ID") {
```

Comment:
Untested but should work.

Reply:
iirc for those ID_MODEL might actaully be correct as it's used for the name rather then being assumped to be an id

Reply:
ok great, i can quickly revert that hunk but i think it makes sense to be sure first by checking a device ?

Reply:
Updated.